telemetry/csharp/TelemetryPackage.proj

```diff
@@ -0,0 +1,13 @@
+<Project ToolsVersion="15.0" DefaultTargets="datatype-pack" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<Target Name="datatype-pack">
```

Comment:
buildspec/csharpTests.yml and .github/workflows/csharp-ci.yaml contain C# CI scripts for this repo. Is it possible to have each of those run this msbuild target at some point after the places where "AwsToolkit.Telemetry.sln" is compiled and tested? This would let us know if there are issues in producing the NuGet package.